### PR TITLE
chore: fix maven warnings

### PR DESCRIPTION
**Pull request by sverrehu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -71,6 +75,7 @@
                 <configuration>
                     <finalName>${project.artifactId}</finalName>
                     <appendAssemblyId>false</appendAssemblyId>
+                    <attach>false</attach>
```
Comment (erikgb) [inline pom.xml:78]: I suggest we instead remove the `appendAssemblyId` above and update the Dockerfile with the new filename: `k3a-lag-exporter-jar-with-dependencies.jar`

**Review by erikgb — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (sverrehu) [reply pom.xml]: Done.

Comment (sverrehu) [reply pom.xml]: Ah, overlooked the "the same name as before" part.

Comment (sverrehu) [reply pom.xml]: Like this, then?

Comment (erikgb) [reply pom.xml]: Yes, lovely! ❤️

**Review by erikgb — APPROVED** [review]

**Pull request merged**